exercises/all-your-base/all_your_base_test.rb

```diff
@@ -0,0 +1,114 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'all_your_base'
+
+class AllYourBaseTest < Minitest::Test
+  def test_single_bit_to_one_decimal
+    assert_equal [1], BaseConverter.new.convert(2, [1], 10)
```

Comment:
It would be good if this was a class method rather than an instance method.
`BaseConverter.convert(2, [1], 10)`